### DISCUSSION TITLE
Remove blue line under Menu.items

### DISCUSSION THREAD

**Discussion by phillippppppo** [ant-design/ant-design#46287] [Q&A]
Hello Guys,

which property is the thick blue underline 1cm under Menu.items, when selected or hovered?

I want to get rid of it.

Thank you!

**Comment by MadCcc** [comment]
Use `antd/dist/reset.css` to reset style from browser

**Reply by phillippppppo** [reply]
Thank you for the answer @MadCcc . I still encounter the problem. 

Here is an image with the current problem:

<img width="83" alt="image" src="https://github.com/ant-design/ant-design/assets/121042019/8873eea1-a203-41ff-a623-9a6a3fe8484e">

but when using antd/dist/reset.css it´s still appearing like this.

Thank you a lot in advance!

**Comment by griffinlorimer** [comment]
I was able to get remove the bottom border using 
```
.ant-menu-item-selected::after, .ant-menu-item:hover::after{
    border-bottom: none !important;
}
```

**Reply by ayyanar-7** [reply]
Thank you for providing solution
